app/views/coc/change_in_earnings_recipient.scala.html

```diff
@@ -43,16 +43,16 @@ <h2 class="heading-medium">@Html(Messages("change.status.earnings.h2"))</h2>
     </li>
 </ul>
 
-<h3 class-"heading-small">@Html(Messages("hmrc.contact-details-1"))</h3>
-<p>@Html(Messages("hmrc.contact-details-2")) <br>
-    @Html(Messages("hmrc.contact-details-3")) <br>
-    @Html(Messages("hmrc.contact-details-4"))</p>
+<h3 class-"heading-small">@messages("hmrc.contact-details-1")</h3>
+<p>@messages("hmrc.contact-details-2") <br>
+    @messages("hmrc.contact-details-3") <br>
+    @messages("hmrc.contact-details-4")</p>
 
-<p>@Html(Messages("hmrc.contact-details-5")) <br>
-    @Html(Messages("hmrc.contact-details-6"))</p>
+<p>@messages("hmrc.contact-details-5") <br>
+    @messages("hmrc.contact-details-6")</p>
 
-<p>@Html(Messages("hmrc.contact-details-7")) <br>
-    @Html(Messages("hmrc.contact-details-8"))</p>
+<p>@messages("hmrc.contact-details-7") <br>
+    @messages("hmrc.contact-details-8")</p>
 
-<p>@Html(Messages("hmrc.contact-details-9"))</p>
+<p><a href=@callChargeUrl>@messages("hmrc.contact-details-9")</a></p>
```

Comment:
So your other changes to the `hmrc.contact-details-9` to add the link in aren't wrapped in a P tag like this one. After speaking with @ma3574, we should wrap the others in the P tag as an anchor tag is an inline element whereas p tag is a block element

Reply:
Hi @danewatts I think you might have tagged the wrong person 😄 

I got a notification for this but have no idea what any of this is 👍

Reply:
lol sorry @ma3574, @Muhammad-Ali-Ahmad is who I meant :man_facepalming: